dashboard/app/models/levels/level.rb

```diff
@@ -95,6 +95,7 @@ class Level < ApplicationRecord
     start_libraries
     ai_tutor_available
     offer_browser_tts
+    use_secondary_finish_button
```

Comment:
Since this is pretty HOC/Music Lab specific, thoughts on just including this in music level data to keep the field super localized to Music Lab? This approach is fine too though if we expect to use this field for other labs

Reply:
I figure that since the implementation is all under `src/lab2/` that it's reasonable to have it (almost) available for other labs too.